### DISCUSSION TITLE
Adding second extruder to move menu

### DISCUSSION THREAD

**Discussion by deusrex2k** [Jyers/Marlin#1823] [Q&A]
I'm in the process of updating Jyers to work with 2 extruders (2in 1out hotend). I think I understand the Marlin updates (pins.h and configuration.h), but I'm not sure what to do in DWIN.cpp to add the second extruder to the Move menu. Can someone send a general pseudo code for updating the menu?

**Comment by antwal** [comment]
I have the same problem, I enabled the second extruder using a single nozzle, from gcode the change works perfectly but from the menu this option is not available and I cannot enable it.

I tried with the basic marlin and in the menu the second extruder is enabled and can be controlled; I think the problem depends on the "jyers" graphics
